tests/osaka/eip7692_eof_v1/eip5450_stack/test_code_validation.py

```diff
@@ -561,9 +561,67 @@ def test_all_opcodes_stack_underflow(
                 ),
             ],
         ),
+        Container(
+            name="underflow_variable_stack_3",
+            sections=[
+                Section.Code(
+                    Op.RJUMPI[8](0)
+                    + Op.PUSH0 * 2
+                    + Op.RJUMPI[1](0)
+                    + Op.POP * 2
+                    + Op.PUSH0 * 2
+                    + Op.REVERT,
+                    max_stack_height=4,
+                ),
+            ],
+        ),
     ],
     ids=lambda x: x.name,
 )
 def test_stack_underflow_examples(eof_test, container):
     """Test EOF validation failing due to stack underflow at basic instructions."""
     eof_test(container=container, expect_exception=EOFException.STACK_UNDERFLOW)
+
+
+@pytest.mark.parametrize("initial_stack", [0, 1, 2])
+@pytest.mark.parametrize("calldata", [(0, 0), (0, 1), (1, 0), (1, 1)])
```

Comment:
```suggestion
@pytest.mark.parametrize("calldata_1", [0, 1])
@pytest.mark.parametrize("calldata_2", [0, 1])
```
It's a bit more straight-forward.